examples/CREATE_AND_USE_A_SESSION.md

```diff
@@ -0,0 +1,86 @@
+### Create and Use a Session
+
+| Key                                                                                         | Description           |
+| ------------------------------------------------------------------------------------------- | --------------------- |
+| [sessionConfigs](https://bcnmy.github.io/biconomy-client-sdk/types/BuildUserOpOptions.html) | CreateSessionConfig[] |
+
+```typescript
+import {
+  createSmartAccountClient,
+  createSession,
+  createSessionSmartAccountClient,
+} from "@biconomy/account";
+import { createWalletClient, http, createPublicClient } from "viem";
+import { privateKeyToAccount, generatePrivateKey } from "viem/accounts";
+import { mainnet as chain } from "viem/chains";
+
+const account = privateKeyToAccount(generatePrivateKey());
+const signer = createWalletClient({ account, chain, transport: http() });
+const smartAccount = await createSmartAccountClient({
+  signer,
+  bundlerUrl,
+  paymasterUrl,
+}); // Retrieve bundler and pymaster urls from dashboard
+const smartAccountAddress = await smartAccount.getAccountAddress();
+
+const { sessionKeyAddress, sessionStorageClient } =
+  await createAndStoreNewSessionKey(smartAccount, chain);
+
+const { wait, session } = await createSession(
```

Comment:
Is there a situation where passed smartAccount and sessionKeyAddress arguments would be different then the ones calculate above ? If not, we could improve this even more by removing the need to pass smartAccount and sessionKeyAddress in the "createSession" method. You could bind the method to the smartAccount like `await smartAccount.createSession([{contractAddress: "0x...", functionSelector: "safeMint(address)", rules: ....}], sessionStorageClient, {paymasterServiceData ...})`

Reply:
This was for a technical reason, at some point soon I want to move sessions into it's own repo because it imports merkletree. If I add the 'createSession' method to the smartAccount then the session code would need to live in the accounts package permanently, like paymaster and bundler etc. Keeping this setup will facilitate the future removal of the relatively large sessions package from our accounts package.

Reply:
this is specific to creating a session struct using ABI svm?

Reply:
and it will also send tx on-chain to set session using current active validation module(default ecdsa) ?

Reply:
Yes this is for ABI svm. Just using a more accessible term. Yes it encorporates the tx to change the validation module, if required

Reply:
the userop that gets built for this only supports single tx? setting one session struct on-chain on SKM.
or calldata batch is enabled where you set multiple sessions and also get session ids back in return along with tx hash. 

using via multi-chain validation module goes one step further where you can send multiple userops (each op can have multiple session enabling as part of calldata batch on particular chain) that enables several same session on multiple chains